samples/DispatcherSample.Web/CorsApis.cs

```diff
@@ -0,0 +1,666 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using Microsoft.Net.Http.Headers;
+
+namespace DispatcherSample.Web
```

Comment:
Since this is just an architectural sample, does it make sense to just do something cheezy rather than copying all of the CORS code into this repo?

To me the value of the sample is that we have an example of the architecture to point to, the fact that it's a real CORS implementation isn't the main idea.

Reply:
Sure, I agree. Actually I didn't think about checking this sample in and just wanted to see how things would work with real Cors implementation. But yeah, we could have this in sample and have couple of tests to verify a integration like scenario.

Reply:
Yeah, I would suggest trying to keep it limited.